src/test/java/it/configuration/SymConfigTest.java

```diff
@@ -0,0 +1,73 @@
+package it.configuration;
+
+import configuration.SymConfig;
+import configuration.SymConfigLoader;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.InputStream;
+
+import static org.junit.Assert.*;
+
+public class SymConfigTest {
+
+    private SymConfig config;
+
+    @Before
+    public void setUp() {
+        InputStream configStream = SymConfigTest.class.getResourceAsStream("/bot-config.json");
```

Comment:
Was the bot-config.json modified for the tests? If so, please put the changes.

Reply:
No, I did not modify the bot-config.json because if I do so it will break all the others test.

Reply:
I use this before each test to set the context path `config.setContextPath("ctx");`